spec/controllers/attendances_controller_spec.rb

```
@@ -1,73 +1,42 @@
-# is no simpler way to get a handle on the object needed for the example.
-# Message expectations are only used when there is no simpler way to specify
-# that an instance is receiving a specific message.
-#
-# Also compared to earlier versions of this generator, there are no longer any
-# expectations of assigns and templates rendered. These features have been
-# removed from Rails core in Rails 5, but can be added back in via the
-# `rails-controller-testing` gem.
-
 RSpec.describe AttendancesController, type: :controller do
+  login_user
+
+  let(:user) { create(:user)} 
+  let(:meeting) { create(:meeting) }
 
-  # This should return the minimal set of attributes required to create a valid
-  # Attendance. As you add validations to Attendance, be sure to
-  # adjust the attributes here as well.
   let(:valid_attributes) {
-    skip("Add a hash of attributes valid for your model")
+    {
+      meeting_id: meeting.id,
+      user_id: user.id,
+      special_duties: 'special duties',
+      completed_hours: 20
+    }
   }
 
   let(:invalid_attributes) {
-    skip("Add a hash of attributes invalid for your model")
+    {
+      meeting_id: nil,
+      user_id: user.id,
+      special_duties: '',
+      completed_hours: nil
+    }  
   }
 
-  # This should return the minimal set of values that should be in the session
-  # in order to pass any filters (e.g. authentication) defined in
-  # AttendancesController. Be sure to keep this updated too.
-  let(:valid_session) { {} }
-
   describe "GET #index" do
     it "returns a success response" do
-      Attendance.create! valid_attributes
-      get :index, params: {}, session: valid_session
-      expect(response).to be_successful
-    end
-  end
 
-  describe "GET #show" do
-    it "returns a success response" do
-      attendance = Attendance.create! valid_attributes
-      get :show, params: {id: attendance.to_param}, session: valid_session
+      Attendance.create! valid_attributes
```

Good call on using the `!` version here so it'll fail loudly in the right spot 🙌🏼